src/net.h

```diff
@@ -123,7 +123,17 @@ extern bool fListen;
 extern uint64_t nLocalServices;
 extern uint64_t nLocalHostNonce;
 extern CAddrMan addrman;
+
```

Comment:
It's not easy to integrate a 'floating' comment like this into doxygen.
One way would be to group nMaxConnections and nWhiteConnections using `@{` `@}` and add it as doxygen comment of the group.